src/pages/Dashboard/Apps/Detail-bak/index.jsx

```diff
@@ -0,0 +1,699 @@
+import React, { Component, Fragment } from 'react';
```

Comment:
If this file not used, can remove it

Reply:
ok，maybe other undeveloped pages refer to this page.